hijack/helpers.py

```diff
@@ -150,6 +154,6 @@ def redirect_to_next(request, default_url=hijack_settings.HIJACK_LOGIN_REDIRECT_
     is_safe_url_kwargs = {}
     if django.VERSION >= (1, 11):
         is_safe_url_kwargs['allowed_hosts'] = {request.get_host()}
```

Comment:
I suggest removing this code now. And add the parameter on the line below. Since we do not have support for versions less than 1.11

Reply:
I removed it in another pull-request. I would keep it here, since it wouldn't fit in any of the commits.